### DISCUSSION TITLE
Follow previous recorded track

### DISCUSSION THREAD

**Discussion by PacmanForever** [OpenTracksApp/OpenTracks#2026] [General]
Hi everyone and thanks for this app.

Do you know if is there a way to load a previous recorded track and follow it again on the map? That is, so that the pointer of your location appears on the previously recorded route.

Thanks

**Comment by pstorch** [comment]
Start a new track recording. Go back to the track list. Select the previous recorded track plus the new track and select show on map. If you have OSMDashboard installed, you see both and can follow the previous one.

**Reply by PacmanForever** [reply]
WOW!! Thank you very much.
